server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileFactory.java

```diff
@@ -123,9 +125,10 @@ private QProfileDto doCreate(DbSession dbSession, OrganizationDto organization,
   public void deleteByKeys(DbSession dbSession, Collection<String> profileUuids) {
     if (!profileUuids.isEmpty()) {
       db.qualityProfileDao().deleteProjectAssociationsByProfileUuids(dbSession, profileUuids);
-      db.activeRuleDao().deleteParametersByProfileKeys(dbSession, profileUuids);
-      db.activeRuleDao().deleteByProfileKeys(dbSession, profileUuids);
-      db.qProfileChangeDao().deleteByProfileKeys(dbSession, profileUuids);
+      db.activeRuleDao().deleteParametersByProfileUuids(dbSession, profileUuids);
+      db.activeRuleDao().deleteByProfileUuids(dbSession, profileUuids);
+      // FIXME these are not uuids of rules_profiles
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)